packages/frontend/core/src/components/blocksuite/block-suite-editor/blocksuite-editor-container.tsx

```diff
@@ -88,7 +89,7 @@ const useBlockElementById = (
     return () => {
       canceled = true;
     };
-  }, [container, blockId, timeout]);
+  }, [blockId, containerRef, timeout]);
```

Comment:
I guess `containerRef` should be `containerRef.current`
```suggestion
  }, [blockId, containerRef.current, timeout]);
```

Reply:
eslint will warning when `containerRef.current`, there are same, as containerRef.current never change in this the component lifecycle.